src/pages/FAQ.tsx

```diff
@@ -12,7 +12,6 @@ import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
```

Comment:
Also filenames in pages need to be all lowercase, manual change since git doesn't track upper and lower case. Without it, gh pages returns a flash of 404